### PR TITLE
The nyt article

### PR DESCRIPTION
**Pull request by SamanAtashi**
Basic Skeleton of the NYT article has been done.

### CODE REVIEW

**Review by Loosescrew022 — APPROVED** [review]
Looking good did not found anything for fix

**Review by chukwuemeka1234 — COMMENTED** [review]
Nice work saman. I couldn't find anything wrong here

File: index.html [hunk 1]
```diff
@@ -17,19 +17,29 @@
...
-			</div>
-			<div class="top-center">
-				<a href="https://www.nytimes.com/"
-					><img src="img/The_New_York_Times_logo.png" alt="NYT logo"
-				/></a>
-			</div>
-			<div class="top-right">
-				<button>SUBSCRIBE NOW</button>
-				<button>LOG IN</button>
+			<div class="nav-content-wrapper">
```
Comment (danmainah) [inline index.html:20]: I love the code it's clean and easy to follow. Checkout the linter error

**Comment by ahzia** [comment]
Looking Great

**Review by chukwuemeka1234 — COMMENTED** [review]
More work on the footer needed. Good job so far.

**Comment by barackm** [comment]
Nice work, up to now everything looks good for me.

**Comment by SamanAtashi** [comment]
Global HTML,CSS has been done including header,main,footer.

---->(First review)<----

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes requested :red_circle: 

### Hello @SamanAtashi :wave: 

### You have done a great job on this project, your page is looking good and almost the same as the original, congratulations! There's are only some details I need you to work on before moving forward, please check below:

- [ ] You are missing one major part of the original webpage (Suggested newsletter and Most Popular), please check below:

Screenshot from 2021-03-24 11-03-02

- [ ] You don't need to commit the file `.DS_Store`, can you please add it to `.gitignore`?

- [ ] You are using a lot of ID Selectors to style your app, it's preferable that you use class selectors for this as described in the [CSS best practices article](https://speckyboy.com/good-bad-css-practices).

As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 code reviews left for this project.

Kindly make the requested changes and submit them for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by smcommits — CHANGES_REQUESTED** [review]
# Status: Changes Required 
### Hello, @SamanAtashi! You have done a great job. The website appears to be fantastic. Although you have done remarkable work, I can see that you still need to implement some changes requested by the previous TSE.

## Changes Required 

- [ ] **Please put .DS_Store in gitignore**

The file .DS_Store is not required by your project and hence should be put in .gitignore.  A .gitignore file specifies intentionally untracked files that Git should ignore. 

- [ ] **The text and images in "More in Space", "Suggested Articles", and "Editor's Pick" sections need to be a link**

# 

### Please request another review from your dashboard once you have implemented the changes

### As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 1 code review left for this project. If you think that the code review was not fair, you can request a second-opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

### Please reach out to me on Slack if you need any assistance.

**Review by Temmarie — APPROVED** [review]
## Status: Approved :heavy_check_mark:

### Hi @SamanAtashi and @ahzia  ! :wave:

#### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: :tada:
 **Recommended for future purpose**
 _These are suggestions on things to be considered for future purposes:_

 - [ ] You have to write a descriptive Pull Request Message with a descriptive title as well. It's essential that your repository is professional, as this is one of the things that potential colleagues will look at when recruiting team members. Please refer to [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) for more information.
 - [ ] You have a horizontal overflow in your project. Be sure to remove it next time.
 - [ ] Check out [font awesome](https://fontawesome.com/v4.7.0/icons/) to get  icons.
 - [ ] Check out [this](https://chrome.google.com/webstore/detail/font-picker/mmjbimgpcbaegjiieojddickpjbdkeej) chrome extension. It helps to get the font-family, size, and a lot of other properties used on a page.


#### Good luck with your next projects! :rocket:
#### Happy Coding! :+1:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Pull request merged**